CMakeLists.txt

```diff
@@ -96,6 +95,8 @@ else()
     #for Mingw64 support, see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=65782
     if(MINGW)
         list(APPEND flags_to_test -fno-asynchronous-unwind-tables)
+    else()
+        list(APPEND flags_to_test -fstack-protector-strong)
```

Comment:
why it doesn't apply also to MINGW?
Is there some performance sacrifice?

Reply:
As mentioned in https://github.com/OpenVisualCloud/SVT-HEVC/issues/118, some builds of mingw-w64 toolchain does not have the required stuff stack protector. Although some do, but it would be hard to properly detect it in cmake due to the errors appearing during link time

Reply:
Do you have an estimation of performance overhead of using -fstack-protector-strong?

Reply:
As far as I know, stack protector is more of a security thing rather than performance. As far as I know, for systems other than MINGW, the performance should be the same. It is only for when compiling using MINGW, that stack protector is not applied

Reply:
I did some test. Didn't see performance impact. Thx.